docs/source/teams/teams_plugins.rst

```diff
@@ -414,13 +416,17 @@ Open 2 more ssh sessions, and start the webserver and scheduler in each.
 
 **Add the Firewall Rule**
 
-Navigate to the networking / firewall rules section of the google cloud console and allow traffic on that port for the VM.
+Navigate to the networking / firewall rules section of the google cloud console and allow traffic on that port for the
+VM.
 
-Once this is done, you should be able to navigate to the airflow interface at `http://<vm ip>:8080` (or the port you chose) and log in with the credentials you created earlier.
+Once this is done, you should be able to navigate to the airflow interface at `http://<vm ip>:8080` (or the port you
+chose) and log in with the credentials you created earlier.
 
 **Mount the Plugins Directory**
 
-The orchestrator must have the same plugins available to it as the instance which queued the operation. This could be accomplished by either installing the plugins on the orchestrator, or by mounting the plugins directory from the instance which queued the operation.
+The orchestrator must have the same plugins available to it as the instance which queued the operation. This could be
+accomplished by either installing the plugins on the orchestrator, or by mounting the plugins directory from the
+instance which queued the operation.
```

Comment:
I think this is whitespace only change - but shouldn't this mention NFS or something like that?